### PR TITLE
Author Image RN Complete Fix

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#41614

## Description
Author image was not listed as file type that should not appear in the RN. Fixed it by adding it to skipped files from RN

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [x] Documentation
- [x] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Comment by tomneeman151293** [comment]
test failure to indicate bug
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/12128/workflows/425fcdee-3b61-48fb-9248-a8a580f1847c/jobs/62814

**Comment by tomneeman151293** [comment]
FYI @tallieber @ShahafBenYakir

**Comment by tomneeman151293** [comment]
@dantavori If this is possible to be checked today because tomorrow we have SDK release and this is blocking a contribution PR

File: demisto_sdk/commands/validate/tests/validators_test.py [hunk 1]
```diff
@@ -803,7 +804,7 @@ def test_validate_no_missing_release_notes__missing_rn_dependent_on_api_module(s
         id_set_f = tmpdir / "id_set.json"
         id_set_f.write(json.dumps(id_set_content))
         validate_manager = ValidateManager(id_set_path=id_set_f.strpath)
-        modified_files = {api_script1.yml.rel_path}
+        modified_files = {api_script1.yml.path}
```
Comment (dantavori) [inline demisto_sdk/commands/validate/tests/validators_test.py:807]: why is this related to the change?

**Review by dantavori — APPROVED** [review]

File: demisto_sdk/commands/validate/tests/validators_test.py (hunk 1 above)
Comment (tomneeman151293) [reply demisto_sdk/commands/validate/tests/validators_test.py:807]: Noticed it was not actually tested, it was returning None because file did not really exist, so i fixed this test to actually work
